airflow/migrations/versions/0114_2_4_0_add_dataset_model.py

```diff
@@ -94,6 +99,12 @@ def _create_task_outlet_dataset_reference_table():
             name="todr_dataset_fkey",
             ondelete="CASCADE",
         ),
+        sa.ForeignKeyConstraint(
+            columns=('dag_id',),
+            refcolumns=['dag.dag_id'],
+            name='todr_dag_id_fkey',
```

Comment:
Could you make this name match the same pattern we use on the rest of the FKs please.